website/src/categories/components/tree-view/code.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: Tag code
+title: Tree code
 tags: code
 APIdescription: {
-  sl-tag: "Tag has a range of properties to define the experience in different use cases.",
-  sl-tag-list: "Tag list component provides properties to define the experience in different use cases, and it is a kind of wrapper for multiple tags."
+  sl-tag: "Tree has a range of properties to define the experience in different use cases.",
```

Comment:
This is wrong; line below it as well :)

Reply:
I guess this is just a small change for the future 😁 `Tree` documentation is still hidden and marked as `planned` in the documentation website. We haven't worked on the Tree documentation yet, so I guess it will be part of other story, also this one: https://github.com/sl-design-system/components/issues/1806.